src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -249,7 +250,7 @@ class InstrumentCodeAction implements Action<Task> {
      */
     private void compileJava(AntBuilder ant, Set<File> srcDirs, File destDir, String classpath) {
         ant.javac(destdir: destDir.canonicalPath, source: getSourceCompatibility(), target: getTargetCompatibility(),
-                  classpath: classpath, encoding: getEncoding(), executable: getExecutable()) {
+                  classpath: classpath, encoding: getEncoding(), executable: getExecutable(), debug: getCompileWithDebug()) {
```

Comment:
After making this change I found another PR with the same change where you asked for a unit test (https://github.com/bmuschko/gradle-clover-plugin/pull/57#issuecomment-80487256).  I've spent time trying to add a test, and I can't figure out how to do it.  I'm having two issues:
(1) The InstrumentCommandAction isn't added until after the project is evaluated, and I can't figure out how to force that to happen in a unit test
(2) Even if I could, I can't find any way to get access to the action.

Again, I'm happy to add the unit test if you can offer some guidance.